telegram/inline/inputtextmessagecontent.py

```diff
@@ -55,3 +55,5 @@ def __init__(self,
         # Optionals
         self.parse_mode = parse_mode
         self.disable_web_page_preview = disable_web_page_preview
+
+        self._id_attrs = (self.message_text,)
```

Comment:
`text='*bold*', parse_mode='Markdown'`  is arguably not the same as `text='*bold*', parse_mode=None`. Add `parse_mode`?

Reply:
No Id say plain text should be enough here. Doesnt matter how markup changes it in the end on the client.